lizmap/modules/lizmap/lib/Project/ProjectMetadata.php

```diff
@@ -211,6 +222,56 @@ public function getLizmapWebCLientTargetVersion()
         return $this->data['lizmapWebClientTargetVersion'];
     }
 
+    /**
+     * If the QGIS desktop needs an update of the plugin.
+     * The check is not done only if the project has been edited recently compare to the date of the Lizmap Web Client.
+     *
+     * @return bool If the plugin should be updated in QGIS Desktop
+     */
+    public function updateQgisLizmapPlugin()
+    {
+        // Not the best choice, maybe because of git for dev, dates are not correct
+        // $projectDate = date($this->getFileTime());
+        // Better to use lastSaveDateTime in the QGS file
+        $projectDate = strtotime($this->getLastSaveDateTime());
```

Comment:
Perhaps add here a fallback to `getFileTime` if `getLastSaveDateTime` returns no data ?